### ISSUE TITLE
New nightly compiler warnings

### ISSUE THREAD

**Issue by ldm0** [mgeisler/version-sync#103]
e.g.
```rust
pub fn foo() {
    version_sync::assert_markdown_deps_updated!("README.md");
    version_sync::assert_html_root_url_updated!("src/lib.rs");
}
```

Compiler warnings:

```rust
warning: panic message is not a string literal
 --> src/main.rs:3:5
  |
3 |     version_sync::assert_markdown_deps_updated!("README.md");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(non_fmt_panic)]` on by default
  = note: this is no longer accepted in Rust 2021
  = note: this warning originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

warning: panic message is not a string literal
 --> src/main.rs:4:5
  |
4 |     version_sync::assert_html_root_url_updated!("src/lib.rs");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this is no longer accepted in Rust 2021
  = note: this warning originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

warning: 2 warnings emitted
```

Change `panic!(err)` to `panic!("{}", err)` will fix this issue. Check https://github.com/rust-lang/rust/pull/81645.

**Comment by mgeisler** [comment]
Hi @ldm0, thank you very much for the report! This was fixed already by @lopopolo in #95. I'm making a release with the fix as we speak.

**Comment by mgeisler** [comment]
The new version is released: https://crates.io/crates/version-sync/0.9.2 :tada:
